.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Let's go ahead and .gitignore all of these .idea files, a good reminder not to use `git add .` 😄